Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG AZCOPY_VERSION
-ARG GO_VERSION=1.22
+ARG GO_VERSION=1.23.3
```

Comment:
I wonder if we can use `1.23` instead?

Reply:
Good point, I amended the commit accordingly.